### DISCUSSION TITLE
How to create a custom template for serverless

### DISCUSSION THREAD

**Discussion by patrickacioli** [serverless/serverless#10975] [Q&A] [answered]
Hi, 

Guys, In our company we are using serverless for most of services. We develop a guideline about how to structure the repository and every time that we are starting a new service, we need to check the guided lines,. So, I'm searching about how to create a custom template for serverless project, for avoid re-work every time that we start a new project. I can't find any helpful docs. 

It's about define deps, readme, gitignore, folder structure and more..

**Comment by pgrzesik** [comment, marked as answer]
Hello @patrickacioli - there's nothing really needed for creating a serverless template - you can just create a repository that will have the template contents and use it to provision a new service e.g. with `serverless --template-url <url>`. For examples, you can have a look at examples: https://github.com/serverless/examples/ or some community templates like https://github.com/purple-technology/purple-stack
